### PR TITLE
Make navbar disappear after click

### PR DESCRIPTION
**Pull request by ivan-unfolds**

### CODE REVIEW

**Comment by rjmk** [comment]
Looks like this branches off `password-response`, so will review after that is merged

File: src/user/components/nav_bar.js [hunk 1]
```diff
@@ -1,21 +1,40 @@
...
+    const navBar = ReactDOM.findDOMNode(this.refs.navigation_bar)
+    const collapsibleNav = navBar.querySelector('div.navbar-collapse')
+    const btnToggle = navBar.querySelector('button.navbar-toggle')
 
+    navBar.addEventListener('click', (e) => {
+      if (e.target.tagName !== 'A' ||
+          e.target.classList.contains('dropdown-toggle') ||
+          ! collapsibleNav.classList.contains('in')) return
+      btnToggle.click()
+    }, false)
```
Comment (rjmk) [inline src/user/components/nav_bar.js:17]: Isn't this argument `false` by default?

**Comment by rjmk** [comment]
OK, definitely seems kind of hacky, but if this is the way you found to do it, I trust you! Could we take a look at that parameter?

File: src/user/components/nav_bar.js (hunk 1 above)
Comment (ivan-unfolds) [inline src/user/components/nav_bar.js:17]: Apparently it is for most modern browsers but is not in ancient ones. Leave it in or take it out?

**Comment by ivan-unfolds** [comment]
It is definitely hacky. It was quite a common problem so I found a lot of discussion on it. This seemed to be the preferred solution for a lot of people.

File: src/user/components/nav_bar.js (hunk 1 above)
Comment (rjmk) [inline src/user/components/nav_bar.js:17]: Fine to leave in

**Pull request merged**